server/src/routes/bookings.ts

```diff
@@ -34,3 +37,123 @@ bookingRoutes.post(
     });
   }
 );
+
+bookingRoutes.get("/all", async (req: Request, res: Response) => {
+  console.log("GET /bookings/all");
+
+  const bookings = await bookingsData.getAllBookings();
+
+  if (bookings === null) {
+    res.status(404).json({ message : "No Bookings Found"});
+    return;
+  } 
+
+  res.status(200).json(bookings);
+});
+
+bookingRoutes.post("/create", async (req: Request, res: Response) => {
+  console.log("POST /bookings/create");
+
+  new xss.FilterXSS().process(req.body.numOfPeople).trim();
+  new xss.FilterXSS().process(req.body.endTimestamp).trim();
+  new xss.FilterXSS().process(req.body.startTimestamp).trim();
```

Comment:
Shouldn't the timestamps be set equal to the new xss stuff?

Reply:
made changes to go with this